actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/TextBookTocDownloadActor.java

```diff
@@ -0,0 +1,22 @@
+package org.sunbird.learner.actors.bulkupload;
+
+import org.sunbird.actor.core.BaseActor;
+import org.sunbird.actor.router.ActorConfig;
+import org.sunbird.common.request.Request;
+
+import java.io.File;
+
+/**
+ * @author gauraw
+ */
+
+@ActorConfig(tasks = {"textbookTocDownload"}, asyncTasks = {})
+public class TextBookTocDownloadActor extends BaseActor {
+
+    @Override
+    public void onReceive(Request request) throws Throwable {
+        File file =new File("test.csv");
+        file.createNewFile();
+        sender().tell(file, sender());
+    }
+}
```

Comment:
Combine into 1 actor TextbookTocActor, split the functions by checking in the onreceive method